src/components/ConversationSettings/LockingSettings.vue

```diff
@@ -55,6 +67,9 @@ export default {
 	data() {
 		return {
 			isReadOnlyStateLoading: false,
+			writingConversationOptions,
+			writingConversations: writingConversationOptions[this.isReadOnly],
```

Comment:
I think you should rename this and change the logic. It's not a boolean any more but an integer value.

Same in saveWritingConversations, instead of flipping the boolean you should take the value from the multiselect.